changes/21409-fedora-label

```diff
@@ -0,0 +1 @@
+- added builtin label for Fedora Linux
```

Comment:
Let's add somewhere that if the users have a label named `Fedora Linux`, then they need to rename it before running migrations.

Reply:
good call!  added